src/subapps/dataExplorer/ProjectSelector.tsx

```diff
@@ -1,47 +1,60 @@
-import { ProjectResponseCommon } from '@bbp/nexus-sdk';
-import { useNexusContext } from '@bbp/react-nexus';
-import { AutoComplete, Input, notification } from 'antd';
+import { AutoComplete, Input } from 'antd';
 import React, { useState } from 'react';
-import { useQuery } from 'react-query';
+import { makeOrgProjectTuple } from '../../shared/molecules/MyDataTable/MyDataTable';
+import { AggregatedBucket, useAggregations } from './DataExplorerUtils';
 import './styles.less';
+import { CloseOutlined, SearchOutlined } from '@ant-design/icons';
+import { normalizeString } from '../../utils/stringUtils';
 
 interface Props {
   onSelect: (orgLabel?: string, projectLabel?: string) => void;
 }
 
 export const ProjectSelector: React.FC<Props> = ({ onSelect }: Props) => {
-  const { data: projects } = useProjects();
-  const [searchTerm, setSearchTerm] = useState('');
+  const { data: aggregations } = useAggregations();
+  const [showClearIcon, setShowClearIcon] = useState(false);
 
   const allOptions = [
     { value: AllProjects, label: AllProjects },
-    ...(projects?.map(projectToOption) ?? []),
-  ].filter(project => project.value.includes(searchTerm));
+    ...(aggregations?.projects.buckets.map(projectToOption) ?? []),
+  ];
 
   return (
     <div className="form-container">
       <span className="label">Show me </span>
```

Comment:
the label of the selector: `Project`

Reply:
My gosh :sweat_smile: thanks for catching that.